tests/test_validator_integration.py

```
@@ -0,0 +1,1014 @@
+        }
+
+        # config_cli: should pass
+        with tempfile.NamedTemporaryFile(mode="w", suffix=".yaml", delete=False) as f:
+            yaml.dump(gliner_format, f)
+            gc_path = f.name
+
+        try:
+            result = load_and_validate_config(
+                gc_path, full_or_lora="full", method="span", validate=True,
+            )
+            config_ok_with_gc_format = result.report.is_valid
```

Ruff (0.15.1)

[error] Local variable `config_ok_with_gc_format` is assigned to but never used

Remove assignment to unused variable `config_ok_with_gc_format`

(F841)